quattor/types/aquilon.pan

```diff
@@ -29,10 +29,18 @@ type structure_cluster = {
     "down_hosts_threshold" ? long(0..)
     "node_index" ? long(0..)
 };
- 
+
+type structure_archetype_os = {
+    @{ Name of OS (OperatingSystem.name) }
+    "name" : string
+    @{ Version of OS (OperatingSystem.version) }
```

Comment:
What does OperatingSystem.version mean here? Is this a standard reference to which I am not familiar? Is this a vendor release number or an internal release? Is there any guidance on how to set or use this? e.g. should it match the --osversion field in Aquilon?